.github/ISSUE_TEMPLATE.md

```diff
@@ -4,13 +4,21 @@
 2. 
 3. 
 
+<!--
+You may drag & drop the attachement (e.g. repro, screenshot) onto the issue.
+-->
+
 ## Current Behavior
 
+<!--
 What is the current behavior?
+-->
```

Comment:
What about we just remove this and the "expected behavior" description below? It doesn't add anything which isn't clear from the section header anyway and would make the template simpler.

Reply:
My worry is that then people would just skip the section